### ISSUE TITLE
Face Recoginition using VGG16

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#471]

**Comment by NEERAJAP2001** [comment]
Description:

Briefly, the VGG-Face model is the same NeuralNet architecture as the VGG16 model used to identity 1000 classes of objects in the ImageNet competition. The VGG16 name simply states the model originated from the Visual Geometry Group and that it was 16 trainable layers. The main difference between the VGG16-ImageNet and the VGG-Face model is the set of calibrated weights as the training sets were different.

The model architecture is a linear sequence of layer transformations of the following types:

* Convolution + ReLU activations
* MaxPooling
* Softmax

Resource: [Link](https://www.robots.ox.ac.uk/~vgg/research/very_deep/)

@akshitagupta15june I'd like to work on this issue.
Please assign this to me.
Thanks

**Comment by akshitagupta15june** [comment]
assigning
